tests/rest_api/setup

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
This setup file can only be applied on Linux. This makes it impossible to develop tests for cloud storage from Windows/MacOS machine. I see few ways for resolving this problem:

1) Fast: mentioning about such inconvenience in README

2) May take a long time: find out a solution that can be applied on Linux, MacOS and on Windows as well.

3) Compromise between previous two: provide instructions for creating local MINIO buckets on different operating systems.